pkg/build/controller/build/build_controller.go

```diff
@@ -2190,17 +2190,25 @@ func (bc *BuildController) createBuildSignaturePolicyData(config *configv1.Image
 		}
 	}
 
+	// Local containers-storage transport should always allow image "pull"
+	acceptAnyScope := make(signature.PolicyTransportScopes)
```

Comment:
I think `storageScope` would be a better name here.  And then the fact that it `acceptsAny` is a function of the use of `signature.NewPRInsecureAcceptAnything()`

if we ever had a need to qualify containers-storage like we do for the two transports ones around atomic and docker above, the name still makes sense

so we end up `transportScopes` and `storageScope`